src/stcal/dark_current/dark_class.py

```diff
@@ -1,4 +1,5 @@
 import numpy as np
+from astropy import units as u
```

Comment:
Is this a standard import idiom for astropy units?  If not, using just 'u' may be too terse.

Reply:
It is: https://docs.astropy.org/en/stable/units/index.html